packages/manager/modules/pci/src/projects/project/storages/cold-archive/add/components/steps/link-user-archive/controller.js

```diff
@@ -209,34 +209,41 @@ export default class ColdArchiveLinkUserArchiveController {
     this.userModel.linkedMode.credential = null;
   }
 
-  onLinkedUserClicked() {
+  onLinkedUserClicked(user) {
     this.trackClick(
       `${COLD_ARCHIVE_TRACKING.ADD_USER.ASSOCIATE.EXISTING_USER}::${COLD_ARCHIVE_TRACKING.ACTIONS.CONFIRM}`,
     );
+    if (user.s3Credentials?.length || user.s3Credentials?.access) {
```

Comment:
**question:** Is user an object or an array? 
If it is object in some scenario and array in remaining cases, please change the implementation to maintain consistency.

Reply:
Sure, It will be addressed in the scope of the ticket : https://jira.ovhcloud.tools/browse/MANAGER-10677